src/lighthouse.php

```diff
@@ -83,8 +83,25 @@
     */
 
     'cache' => [
+        /**
```

Comment:
```suggestion
        /*
```

Because they are not actual PHPDocs.